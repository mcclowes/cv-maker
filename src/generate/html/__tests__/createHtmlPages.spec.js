import createHtmlPages from "../createHtmlPages";
import markdown from "./markdown.md";

describe("createHtmlPages", () => {
  it("generates correct html", () => {
    expect(createHtmlPages(markdown)).toMatchSnapshot();
  });
});
