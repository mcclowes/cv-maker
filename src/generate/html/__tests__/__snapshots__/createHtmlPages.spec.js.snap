// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createHtmlPages generates correct html 1`] = `
"<div class=\\"page\\" id=\\"p1\\"><h1 id=\\"max-clayton-clowes\\">Max Clayton Clowes</h1>
<p><a href=\\"mailto:contact@mcclowes.com\\">contact@mcclowes.com</a> · <a href=\\"tel:+447976136097\\">+44 (0) 7976 136 097</a> · <a href=\\"https://mcclowes.com\\">mcclowes.com</a></p>
<h2 id=\\"experience\\">Experience</h2>
<h3 id=\\"software-engineer-fe---pollen\\">Software Engineer (FE) <em>- Pollen</em></h3>
<p><em>December 2019 - Ongoing</em></p>
<p>Developed Pollen’s checkout and payment application (React/Next.js/GraphQL).</p>
<h3 id=\\"software-engineer-fe---workshare\\">Software Engineer (FE) <em>- Workshare</em></h3>
<p><em>August 2018 - November 2019</em></p>
<p>Built Transact, a document management &amp; sign-off web app (React/Redux). Collaborated with design to define roadmap. Trained junior team members and pioneered our best practices.</p>
<h3 id=\\"product-engineer--manager---codogo\\">Product Engineer &amp; Manager <em>- Codogo</em></h3>
<p><em>April 2016 - August 2018</em></p>
<p>Founded Codogo after winning several enterprise awards. Established an agency business, supporting development of our own product, Write. I personally:</p>
<ul>
<li>Designed and delivered products for 30+ clients ranging from the NHS to leading financial think-tanks.</li>
<li>Developed software, websites, web apps, and native apps, utilising a range of modern technologies.</li>
<li>Managed (Agile) and supported the growth of my small team.</li>
<li>Pitched to top investors, including being flown out to Y Combinator based on the strength of the Alpha of our product, Codogo Write.</li>
</ul>
<p></div> <div class=\\"page\\" id=\\"p2\\"></p>
<h3 id=\\"other-experience\\">Other Experience</h3>
<ul>
<li>Development &amp; Design Intern <em>- Akkroo, 2015</em></li>
<li>Design Contractor <em>- Ludifi, 2014</em></li>
<li>Design Contractor <em>- 4J Studios, 2013</em></li>
<li>Junior UX Designer <em>- Better Brief, 2012 - 2013</em></li>
<li>Project manager <em>- Godfreys.co, 2011 - 2012</em></li>
</ul>
<h2 id=\\"education\\">Education</h2>
<h3 id=\\"bsc-hons-computer-science-21---durham-university\\">BSc (Hons) Computer Science, 2:1 <em>- Durham University</em></h3>
<p><em>2012 - 2016</em><br>British Computing Society accredited. Awarded Rising Star of Research.<br>Dissertation: <em>The use of evolutionary AI in the balancing of competitive multiplayer games</em></p>
<h3 id=\\"ba-interaction-design-year-1---ravensbourne-college-of-design\\">BA Interaction Design (Year 1) <em>- Ravensbourne College of Design</em></h3>
<p><em>2011 - 2012</em></p>
<h3 id=\\"foundation-diploma-in-art-and-design---london-college-of-communication\\">Foundation Diploma in Art and Design <em>- London College of Communication</em></h3>
<p><em>2010 - 2011</em><br>Graphic Design pathway</p>
<h3 id=\\"high-school---city-of-london-school\\">High School <em>- City of London School</em></h3>
<p><em>2003 - 2010</em><br>A levels: <em>Physics, Economics, Music</em></p>
<h3 id=\\"other-training\\">Other Training</h3>
<ul>
<li>Project Management <em>- SuperHi, 2019</em></li>
<li>Advanced iOS Development <em>- City University, 2012</em></li>
<li>Adva nced C Programming <em>- City University, 2011</em></li>
</ul>
<h2 id=\\"about-me\\">About Me</h2>
<p>Regularly attend PM conferences &amp; meet-ups <em>(Elevate, Product School)</em>. Regularly attend developer meet-ups <em>(React London, GitHub Universe, Next Conf)</em>. Currently work mainly with React <em>(ES6)</em>.</p>
<p>Runner (1st marathon coming up in December), cyclist, rock climber, and dancer (1st place university circuit). Singer and multi-instrumentalist.</p>
</div>"
`;
